ibis/backends/tests/test_register.py

```diff
@@ -476,7 +465,11 @@ def test_read_parquet_glob(con, tmp_path, ft_data):
     for fname in fnames:
         pq.write_table(ft_data, tmp_path / fname)
 
-    table = con.read_parquet(tmp_path / f"*.{ext}")
+    if con.name == "clickhouse":
+        # clickhouse does not support read directory
+        table = con.read_parquet(tmp_path / f"*.{ext}")
```

Comment:
This doesn't seem like the right approach. You're changing what's being tested. Why can't you leave this code unchanged here?

Reply:
the pyarrow `read_table` cannot read things like `tmp_path/*.parquet` and clickhouse cannot read the `directory`

Reply:
The test is whether the backend can read a glob of parquet files, the answer to that seems to be "no", so it should be marked as `notyet`